cliquet_docs/reference/permission.rst

```diff
@@ -1,102 +1,188 @@
+| GET / HEAD | /{collection}      | ``read``             | If not allowed by setting ``cliquet.{collection}_read_principals``,         |
+|            |                    |                      | will return list of records where user has ``read`` permission.             |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
+| POST       | /{collection}      | ``create``           | Allowed by setting ``cliquet.{collection}_create_principals``               |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
+| DELETE     | /{collection}      | ``write``            | If not allowed by setting ``cliquet.{collection}_write_principals``,        |
+|            |                    |                      | will delete the list of records where user has ``write`` permission.        |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
+| GET / HEAD | /{collection}/{id} | ``read``             | If not allowed by setting ``cliquet.{collection}_read_principals``,         |
+|            |                    |                      | will check record permissions                                               |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
+| PUT        | /{collection}/{id} | ``create`` if record | Allowed by setting ``cliquet.{collection}_create_principals``,              |
+|            |                    | doesn't exist,       | or ``cliquet.{collection}_create_principals`` or existing record permissions|
+|            |                    | ``write`` otherwise  |                                                                             |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
+| PATCH      | /{collection}/{id} | ``write``            | If not allowed by setting ``cliquet.{collection}_write_principals``,        |
+|            |                    |                      | will check record permissions                                               |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
+| DELETE     | /{collection}/{id} | ``write``            | If not allowed by setting ``cliquet.{collection}_write_principals``,        |
+|            |                    |                      | will check record permissions                                               |
++------------+--------------------+----------------------+-----------------------------------------------------------------------------+
```

Comment:
nit: is it possible to have a table that fits in less width (review will be made easier). If not or too complicated, don't bother.